internal/commands/open.go

```diff
@@ -51,10 +52,10 @@ func OpenStartIncidentDialog(client bot.Client, triggerID string) error {
 
 	meeting := &slack.TextInputElement{
 		DialogInput: slack.DialogInput{
-			Label:       "War Room URL",
-			Name:        "war_room_url",
+			Label:       "Incident Room URL",
+			Name:        "incident_room_url",
 			Type:        "text",
-			Placeholder: "War Room URL eg. Matrix/Meeting/Zoom",
+			Placeholder: "Incident Room URL eg. Zoom Join Url, Google Meet Url",
```

Comment:
These placeholders are good candidates to be defined via env variables so we don't use wildlife-specific terms here.

But this can be done afterward in a different task.

Reply:
nit: just to keep our OCD under control, let's try to follow a single pattern for acronyms (ex: URL instead of Url)